### PR TITLE
dev2

### PR DESCRIPTION
**Pull request by luismoser99**

### CODE REVIEW

File: Coding/server/src/WiFiManager/WiFiManager.h [hunk 1]
```diff
@@ -101,36 +97,36 @@ class WiFiManager
 
 private:
     /** Reference to store */
-    Store &m_store = Store::getInstance();
+    Store &m_store;
```
Comment (0ooYoo0) [inline Coding/server/src/WiFiManager/WiFiManager.h:100]: Please use a unified order within the header files

**Review by 0ooYoo0 — CHANGES_REQUESTED** [review]

File: Coding/server/lib/FileManager/FileManager.cpp [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+/**
+ * @file FileManager.cpp
+ * @author Luis Moser
+ * @brief FileManager class
+ * @date 07/07/2021
+ * 
+ * @{
+ */
+
+#include <FileManager.h>
```
[the remark this answers is not in the document]
Comment (luismoser99) [reply Coding/server/lib/FileManager/FileManager.cpp]: Fixed with 5927fd3

File: Coding/server/lib/FileManager/FileManager.h [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ * @author Luis Moser
+ * @brief FileManager header
+ * @date 07/07/2021
+ * 
+ * @{
+ */
+
+#ifndef __FILEMANAGER_H__
+#define __FILEMANAGER_H__
+class FileManager
```
[the remark this answers is not in the document]
Comment (luismoser99) [reply Coding/server/lib/FileManager/FileManager.h:45]: Fixed with 08608e1

File: Coding/server/lib/KeyCert/KeyCert.cpp [hunk 4]
```diff
@@ -44,7 +44,8 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 uint8_t KeyCert::m_binaryKey[RSA_KEY_SIZE_BYTE] = {0};
 uint8_t KeyCert::m_binaryCert[CERT_SIZE_BYTE] = {0};
 
-KeyCert::KeyCert()
+KeyCert::KeyCert() : m_sslCert(),
+                     m_certGenerated(false)
 {
 }
```
[the remark this answers is not in the document]
Comment (luismoser99) [reply Coding/server/lib/KeyCert/KeyCert.cpp:52]: Fixed with 08608e1

File: Coding/server/src/WiFiManager/WiFiManager.h (hunk 1 above)
Comment (luismoser99) [reply Coding/server/src/WiFiManager/WiFiManager.h]: Fixed

File: Coding/server/src/HTTPsWebServer/HTTPsWebServer.h [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+    static const uint8_t MAX_CLIENTS = 4;
+
+    /** HTTPSServer instance */
+    httpsserver::HTTPSServer m_httpsServer;
+
+    /** Stores all HTTP routes */
+    httpsserver::ResourceNode m_homeRoute;
+
+    /** Store instance */
+    Store &m_store = Store::getInstance();
```
[the remark this answers is not in the document]
Comment (luismoser99) [reply Coding/server/src/HTTPsWebServer/HTTPsWebServer.h]: Fixed

**Review by 0ooYoo0 — APPROVED** [review]

**Pull request merged**